### ISSUE TITLE
Please add missing transition keywords

### ISSUE THREAD

**Issue by bparadie** [rtfeldman/elm-css#234]
In https://github.com/rtfeldman/elm-css/issues/11 it sounds like somebody needs to express interest in having animation keywords added to `elm-css`. Here we go...

For my project I need:
- [ ] transition
- [ ] transition-delay
- [ ] transition-duration
- [ ] transition-property
- [ ] transition-timing-function

@see https://github.com/rtfeldman/elm-css/issues/11

**Comment by robrtle** [comment]
Here is a stop-gap that I created. Its not likely to be integrated.. But might be useful for you

https://github.com/robrtle/elm-css/blob/transition/src/Css/Transitions.elm

It only has transition but can do everything you might want.  

```
import Css.Transitions as T exposing (transition)
import Time exposing (second)

t : Mixin
t =
    transition
        [ T.height (second * 2)
        , T.width3 (second) (second) T.ease
        ]
```

It would work as a separate package if I packaged it.

**Comment by BobWhitelock** [comment]
I was considering having a go at implementing this, however I notice that the `transition` keywords have 'Working Draft' status; for example see https://developer.mozilla.org/en-US/docs/Web/CSS/transition. 

@rtfeldman is this sufficiently stable to add to elm-css? Not sure what the cutoff is for how stable things should be to be added.

**Comment by anagrius** [comment]
@bobwhitelock I have been using it for several years now, and I would say it is very widely used. I vote to implement it asap.

**Comment by rtfeldman** [comment]
Looks widely supported: http://caniuse.com/#feat=page-transition-events - seems fine to add to me, if anyone wants to make a PR!

**Comment by aleehedl** [comment]
I opened a pull request based on the work done by @robrtle, see #324.

**Comment by xdumaine** [comment]
@robrtle I tried to bring your stop-gap into my project, but found that CSS.Style did not exist. Is that in a recent version of elm-css? I'm stuck on 8.2.0 for the moment.

**Comment by rtfeldman** [comment]
Released in `14.0.0`! Thanks, all!
